tests/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
This extra newline at the start of the file annoys my sense of order. :laughing: